scripts/target/mame/arcade.lua

```diff
@@ -3182,8 +3182,8 @@ files {
 	MAME_DIR .. "src/mame/machine/segabb.h",
 	MAME_DIR .. "src/mame/machine/megadriv.cpp",
 	MAME_DIR .. "src/mame/includes/megadriv.h",
-	MAME_DIR .. "src/mame/drivers/megadrvb.cpp",
-	MAME_DIR .. "src/mame/includes/megadrvb.h",
+	MAME_DIR .. "src/mame/drivers/megadriv_arcade_clones.cpp",
+	MAME_DIR .. "src/mame/includes/megadriv_arcade_clones.h",
```

Comment:
Please use more terse source file names.

Reply:
I think the more verbose filenames help in this case, that's why I'm using them

Reply:
I'd prefer to put them in a megadriv folder, if that's an option, but that seems to be a non-standard approach.

Reply:
I'd just name it megadriv_bootleg.cpp. Let's say for example you'd find a home version of a standalone game Megadrive bootleg, it can be placed in this driver too(instead of making megadriv_home_clones just because it doesn't have a coin slot?)

Reply:
Much like system/software shortnames, we have to work with source file names.  Making them excessively long makes them annoying to type, increases the width of columns in multi-column listings, reduces the number of columns that can be shown in potential completions on the command line, etc.  There's value in being terse.